src/content/docs/tutorial-reporting-data/capture-web-telemetry.mdx

```diff
@@ -0,0 +1,354 @@
+---
+title: "Capture the right web telemetry" 
+metaDescription: "This guide helps you set up the best telemetry possible to describe the runtime operation of your web services."
+redirects:
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/sc-implementation-guide
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/service-characterization-optimize-web-telemetry-guide/
+---
+
+import omaOeScBrowserErrors from 'images/oma-oe-sc_screenshot-browser-errors.webp'
+import omaOeScBrowserErrorLog from 'images/oma-oe-sc_screenshot-browser-errorlog.webp'
+import omaOeScBrowserGroupedPageViews from 'images/oma-oe-sc_screenshot-poor-pageview-grouping.webp'
+import omaOeScBrowserGroupedAjaxRequests from 'images/oma-oe-sc_screenshot-poor-ajax-grouping.webp'
+
+One of the most important things you can do to maintain a high quality production environment is to make sure you have the web telemetry you need to detect and resolve poor user experience. This guide will show you how to check that you're getting the right data and that your browser monitoring is configured for you to make the most of it. We'll help you ensure that you're:   
+
+1. Getting the most value from the data you're collecting
+2. Seeing opportunities to optimize
+3. Able to quickly triage and troubleshoot issues
+4. Getting the data you need to create real-time business KPI dashboards
+
+<Steps>
+    <Step>
+## Tune your browser application naming and sub-account placement [#browser-app-naming] 
+First, you'll need to ensure your browser naming and data organization are in place. If needed, you can change the name of your browser application following the [renaming guide](/docs/browser/new-relic-browser/configuration/rename-browser-apps/). If you have data from multiple environments reporting into one browser application, you can [create new Browser apps](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/) and update the JavaScript snippet in your pages to report the right app. 
+
+Remember to keep the following in mind as you check your browser monitoring organization:
+
```

Comment:
The bullets below this seem funky. Are the second set meant to be nested?

Reply:
Reworded to un-funkify, hopefully this is better